README.md

```diff
@@ -1,2 +1,278 @@
+
+___
+
+## Experiential Learning
+![alt text](https://inteng-storage.s3.amazonaws.com/img/iea/9lwjAVnM6E/sizes/ocde_resize_md.jpg)
+### Essential skills
+###### *These are a list of skills that you can develop on your own*
+
+* Enhance development skills
+  * Have a firm grasp of programming logic and concepts
+    * Object oriented programming, decision structures, repetition structures, data structures
+  * Ensure you have clean coding practice and use design patterns as they are critical for maintenance, reusability and extendability. 
+* Enhance testing skills
+  * This includes being able to balance and switch between the developer and tester mindset that the role requires. 
+* Explore new tools
+  * Not all challenges can be solved with the same tool. Therefore, SDETs must continuously evaluate and add to their collection of tools. 
+  * You must know how to effectively use each tool. 
+  * Analyze the problem and choose which tool is a good fit, rather than solving the problem several times with different tools to find the most effective solution. 
+* Soft skills
+  * Communication 
+  * Leadership
+  * Creativity
+* Keep learning
+  * Take testing to the next level
+  * Keep up with QAs evolution 
+  * Practice, practice, practice
+
+
+### Co-ops
+Through co-op jobs you will be able to practice your knowledge and skills. It will give you the opportunity to apply this knowledge in the real world and help you develop and enhance both your technical and non-technical skills.
+
+Many universities offer co-op programs that allow you to gain real work experience while studying. In addition you may make $50,000 - $60,000 over five years. 
+
+|Example co-op job titles|Example co-op employers|
+|-----------------------|----------------------|
+|<ul><li> Tools and automation engineering intern </li> <li> Data scientist </li> <li> Mobile developer </li> <li> Software developer </li> <li> iOS developer </li> <li> Production engineering </li> <li> Algorithms engineering </li> </ul>|<ul><li> Mozilla Corporation </li> <li> Bloomberg</li> <li> LinkedIn</li> <li> Waterloo Regional Police Service</li> <li> Element AI</li> <li> ContextLogic</li> </ul>|
+
+
+### Internships
+Internships are also a great way to gain experience. It allows you to apply what you’ve learned in a real-world setting. You’ll be mentored during the experience which allows you to gain insight into what the career entails. Additionally it helps you to build your network. It is important to make connections, as they can provide you with information and help you find jobs in the future. You can also use them as references as they have seen you in action and know your strengths. 
+
+#### Example: [Software Development Engineer Intern] (https://www.peersight.in/job/amazon-web-services-canada-in-software-development-engineer-intern-in-vancouver-bc?utm_campaign=google_jobs_apply&utm_source=google_jobs_apply&utm_medium=organic) (Amazon Web Services Canada, In)
```

Comment:
Formatting issues with this link not rendering properly (url should not be visible)